building/glass/aptbranch.py

```diff
@@ -4,7 +4,10 @@
 import re
 import subprocess
 
```

Comment:
Extra newline?

Reply:
This separates imports for our own files from stdlib imports, as per convention.

Reply:
Woops, yeah.